test/e2e/cloud/nodes.go

```diff
@@ -47,8 +47,9 @@ var _ = SIGDescribe("[Feature:CloudProvider][Disruptive] Nodes", func() {
 		nodeDeleteCandidates := framework.GetReadySchedulableNodesOrDie(c)
 		nodeToDelete := nodeDeleteCandidates.Items[0]
 
-		origNodes, err := e2enode.GetReadyNodesIncludingTaintedOrDie(c)
+		origNodes, err := e2enode.GetReadyNodesIncludingTainted(c)
 		framework.ExpectNoError(err)
+
```

Comment:
Here I didn't replace with `gomega.Expect(err).NotTo(gomega.HaveOccurred())`, since it is called directly in a ginkgo test suit.  Please comment if I should replace `ExpectNoError` with `gomega.Expect(err)`